<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1547</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="receive_screen">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>540</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    background-color: rgb(0, 0, 0);
    color: rgb(255, 255, 255);
    border: 2px solid rgb(237, 51, 59);
    border-radius: 5px;
    padding: 5px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="camera_screen">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>240</width>
     <height>180</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border: 2px solid rgb(0, 0, 255); /* 바깥 테두리 색상을 파란색(Blue)으로 변경 */
border-radius: 5px;
padding: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="HTT">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>650</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(98, 160, 234);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/pan_tool_FILL0_wght400_GRAD0_opsz24.svg</normaloff>icons/pan_tool_FILL0_wght400_GRAD0_opsz24.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="STT">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>650</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(98, 160, 234);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/speech_to_text_FILL0_wght400_GRAD0_opsz24.svg</normaloff>icons/speech_to_text_FILL0_wght400_GRAD0_opsz24.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="tts_btn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>650</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(98, 160, 234);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/text_to_speech_FILL0_wght400_GRAD0_opsz24.svg</normaloff>icons/text_to_speech_FILL0_wght400_GRAD0_opsz24.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGuide">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>650</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(205, 171, 143);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/contact_support_FILL0_wght400_GRAD0_opsz24.svg</normaloff>icons/contact_support_FILL0_wght400_GRAD0_opsz24.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="gestureButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>550</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51, 209, 122);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/phone_in_talk_FILL0_wght400_GRAD0_opsz24.svg</normaloff>icons/phone_in_talk_FILL0_wght400_GRAD0_opsz24.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>620</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Ubuntu; font-weight: bold; font-size: 12pt;</string>
   </property>
   <property name="text">
    <string>화면공유</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>720</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Ubuntu; font-weight: bold; font-size: 12pt;</string>
   </property>
   <property name="text">
    <string>수어모드</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>720</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Ubuntu; font-weight: bold; font-size: 12pt;</string>
   </property>
   <property name="text">
    <string>STT</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>720</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Ubuntu; font-weight: bold; font-size: 12pt;</string>
   </property>
   <property name="text">
    <string>TTS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>720</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Ubuntu; font-weight: bold; font-size: 12pt;</string>
   </property>
   <property name="text">
    <string>Guide</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="sub_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>500</y>
     <width>641</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    background-color: rgb(0, 0, 0);
    color: rgb(255, 255, 255);
    border: 2px solid rgb(237, 51, 59);
    border-radius: 5px;
    padding: 5px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="sub_label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>460</y>
     <width>641</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="guideLabel">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>9</y>
     <width>810</width>
     <height>721</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    background-color: rgb(255, 255, 255);
    color: rgb(255, 255, 255);
    border: 2px solid rgb(150, 150, 150);
    border-radius: 5px;
    padding: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>650</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Saab</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Fast</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="hand_speed">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QSlider::groove:horizontal { 
	background: #C9CDD0;
	height: 50px; 
	border-radius: 4px;
}

QSlider::handle:horizontal { 
	background-color: #51A0D5; 
	width: 50px; 
	height: 50px; 
	border-radius: 25px; 
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Saab</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Slow</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="speed_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>540</y>
     <width>571</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="autoword_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="autoword_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="autoword_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="autoword_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="autoword_5">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>60</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_reset">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 221, 218);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/restart_alt_FILL0_wght400_GRAD0_opsz24.svg</normaloff>icons/restart_alt_FILL0_wght400_GRAD0_opsz24.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="text_label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>550</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(237, 51, 59);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/phone_disabled_FILL0_wght400_GRAD0_opsz24.svg</normaloff>icons/phone_disabled_FILL0_wght400_GRAD0_opsz24.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>620</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Ubuntu; font-weight: bold; font-size: 12pt;</string>
   </property>
   <property name="text">
    <string>연결종료</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="zzoomLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>650</y>
     <width>192</width>
     <height>108</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    background-color: rgb(255, 255, 255);
    color: rgb(255, 255, 255);
    border-radius: 5px;
    padding: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="sub_label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    background-color: rgb(0, 0, 0);
    color: rgb(255, 255, 255);
    border: 2px solid rgb(237, 51, 59);
    border-radius: 5px;
    padding: 5px;
</string>
   </property>
   <property name="text">
    <string>[나] : ~~~~~~~</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sub_label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>[상대] :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboFilter">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>730</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>original</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>canny</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>gray</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>blur</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>concave</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>removebg</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>cartoon</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
